crates/prism-recipe/harness/tests/test_g6_censor_and_clusters.py

```
@@ -0,0 +1,212 @@
+    if not math.isfinite(x) or x <= 0.0 or ref <= 0.0:
+        return 0.0
+    denom = math.log(cap / ref)
+    if abs(denom) < sys.float_info.epsilon:
+        return 0.0
+    return min(1.0, max(0.0, math.log(x / ref) / denom))
+
+
+def _curve(points):
+    return {"probe_curve": [
+        {"step": i, "tokens_seen": t, "wall_s": 1.0, "probe_loss": l}
+        for i, (t, l) in enumerate(points)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)